proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -11211,12 +11203,13 @@ lemma is_refill_ready_alt:
   by (clarsimp simp: refill_ready_def vs_all_heap_simps)
 
 method handle_overrun_loop_fail_simple
-  = (clarsimp simp: handle_overrun_loop_def
-     , rule valid_whileLoop_cond_fail
-     , (fastforce simp: head_time_buffer_def obind_def ogets_def read_sched_context_def
-                        vs_all_heap_simps refill_sufficient_def refill_capacity_def
-                        MIN_BUDGET_nonzero sc_valid_refills_def vs_all_heap_simps
-                 split: option.splits kernel_object.splits if_splits)+)
+  = clarsimp simp: handle_overrun_def,
```

Comment:
again, not sure about these / `local_method` / folding into other methods;
the methods namespace is global, i.e. `print_methods` will print all these methods along with the others